tests/src/JIT/Methodical/tailcall/Desktop/thread-race.cs

```diff
@@ -4,16 +4,24 @@
 
 using System;
```

Comment:
@dotnet/jit-contrib  Can I get someone from the JIT team to sign off on this altered test? It has been timing out on ARM64-checked build. I attempted to reduce the runtime duration without changing the spirit of the test too much.

Reply:
I think https://github.com/dotnet/coreclr/pull/24174 should be reverted with this test change.